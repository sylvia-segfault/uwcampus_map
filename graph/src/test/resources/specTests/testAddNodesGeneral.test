# Create an empty graph
CreateGraph graph4

# Add 4 nodes to graph4
AddNode graph4 node1
AddNode graph4 node2
AddNode graph4 node3
AddNode graph4 node4