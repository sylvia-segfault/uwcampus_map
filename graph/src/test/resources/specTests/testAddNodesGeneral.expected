# Create an empty graph
created graph graph4

# Add 4 nodes to graph4
added node node1 to graph4
added node node2 to graph4
added node node3 to graph4
added node node4 to graph4